api/v1alpha1/test/new.go

```diff
@@ -94,10 +94,12 @@ func NewExtendedDaemonSetReplicaSet(ns, name string, options *NewExtendedDaemonS
 			APIVersion: apiVersion,
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Namespace:   ns,
-			Name:        name,
-			Labels:      map[string]string{},
-			Annotations: map[string]string{},
+			Namespace: ns,
+			Name:      name,
+			Labels:    map[string]string{},
+			Annotations: map[string]string{
+				"unready-pods": "0",
```

Comment:
we could define a more descriptive annotation key like `eds.datadoghq.com/unready-pods`

Reply:
Also this annotation key can be defined as `const string` to be re-use in other part of the code

Reply:
I agree. Thanks for suggesting this.

I will update the code.